robot/rospackages/src/image_scp/config.txt

```diff
@@ -0,0 +1,6 @@
+#Use this as a template for entering user details and destination of files so script doesn't need to be changed.
+#Enter credentials/ path without quotation marks 
+Username replaceWithUse
+Host relaceWithHostDomain
```

Comment:
while host domain is probably still correct, it's a bit general. This is supposed to be the OBCs IP address right? Maybe we can just put `roverIP` instead?